text/0233-opentelemetry-sandbox.md

```diff
@@ -0,0 +1,92 @@
+
+The advantage of a sandbox organization is that OpenTelemetry's governance can be used, including Governance Committee (GC) and Technical Committee (TC) intermediation of conflicts, making sure it’s an inclusive place for people to collaborate while keeping the reputation of the OpenTelemetry project as a whole untouched, given that it would be clear that OpenTelemetry doesn’t officially support projects within the sandbox.
+
+There is a desire, but not an expectation, that projects will be moved from the sandbox as an official SIG or incorporated into an existing SIG. Realistically, we know that experiments might get dropped. There’s also no expectation that the OpenTelemetry project will provide resources to the sandbox project, like extra GitHub CI minutes or Zoom meeting rooms, although we might evaluate individual requests.
+
+This OTEP is inspired by [CNCF’s sandbox projects](https://www.cncf.io/sandbox-projects/), but the process is significantly different.
+
+## Internal details
+
+### Acceptance criteria
+
+A low barrier to entry would be desired for the sandbox. While the process can be refined based on our experience, the initial proposal for the process is the following:
+
+1. Proposals should be written following the template below and have one TC and/or GC sponsor, who will regularly provide the TC and GC information about the state of the project.
+2. Once a sponsor is found, the TC and GC will vote on accepting this new project on the Slack channel #opentelemetry-gc-tc.
+    1. After one week, the voting closes automatically, with the proposal being accepted if it has received at least one :thumbs-up: (that of the sponsor, presumably).
+    2. If at least one :thumbs-down: is given, or a TC/GC member has restrictions about the project but hasn’t given a :thumbs-down:, the voting continues until a majority is reached or the restrictions are cleared.
+    3. The voting closes automatically once a simple majority of the TC/GC electorate has chosen one side.
+3. Proponents should abide by OpenTelemetry’s Code of Conduct (currently the same as CNCF’s).
+4. There’s no expectation that small sandbox projects will have regular calls, but there is an expectation that all decisions will be made in public and transparently.
+5. Sandbox projects do NOT have the right to feature OpenTelemetry’s name on their websites.
+
+Initially, there are three slots for sandbox projects available. The GC and TC might vote to increase this number based on the experience with the first projects.
+
+#### Template
+
+> Project name:
+>
+> Repository name:
+>
+> Problems the project will solve:
+>
+> Motivation for joining the sandbox:
+>
```

Comment:
```suggestion
>
> Exit Criteria
>
```

Below you describe possible outcomes for the annual evaluation. i am wondering if the project should be founded with exit in mind, this way it also help existing SIGs to express potential concern (e.g. a SIG might not be able to incorporate without certain criteria met, or a SIG might have an issue with a sandbox forming a new SIG that is relatively close to what they are working on, etc.)

Reply:
That's an interesting idea. I'm not 100% sure yet I agree yet, as there might be experiments without a clear idea of where it'll eventually lead to, but I like this idea.

Reply:
I think putting "We don't know yet" or "this is an experiment with no clear outcome yet", maybe this can be optional?

Reply:
Sounds good, I added the Exit Criteria there now.

Reply:
TY